user-profiles.md

```diff
@@ -0,0 +1,36 @@
+# User Profiles
+
+The purpose of this document is to aid in the development of Helm features by evaluating Helm features against those who will be using them.
+
+When considering requirements and implementation solutions it's useful look at who uses Helm to have a better understanding of how something should work. The Helm user profiles describe the different types of users and contributors to Helm along with the order of priority they have relative to each other. The ordering is because nothing can share the exact same priority and we want to convey that.
```

Comment:
*When considering requirements and implementation solutions/,/ it's useful /to/ look*

Also being nitpicky here, but while in this section:

*different types of users and contributors to Helm/,/ along with the order of priority*